### PR TITLE
Dynamic PROVIDER_VERSIONS_TABLE_NAME name

### PR DESCRIPTION
**Pull request by eranelbaz**
fix #62

### CODE REVIEW

File: src/lambda/populate_provider_versions/store_provider_listing.go [hunk 1]
```diff
@@ -11,9 +11,8 @@ import (
 	"time"
 )
 
-const tableName = "provider-versions"
-
 func storeProviderListingInDynamo(providerNamespace string, providerType string, versions []providers.Version) error {
+	tableName := os.Getenv("PROVIDER_VERSIONS_TABLE_NAME")
```
Comment (eranelbaz) [inline src/lambda/populate_provider_versions/store_provider_listing.go:15]: Cannot be const anymore
```
Const initializer 'os.Getenv("PROVIDER_VERSIONS_TABLE_NAME")' is not a constant
```

Comment (Yantrio) [inline src/lambda/populate_provider_versions/store_provider_listing.go:15]: make sure to check that it exists and is populated

Comment (eranelbaz) [reply src/lambda/populate_provider_versions/store_provider_listing.go:15]: 5f1a9b76(check for non empty)

**Review by Yantrio — APPROVED** [review]

File: lambda.tf [hunk 2]
```diff
@@ -82,6 +82,7 @@ resource "aws_lambda_function" "populate_provider_versions_function" {
 
   environment {
     variables = {
+      PROVIDER_VERSIONS_TABLE_NAME = aws_dynamodb_table.provider_versions.name
```
Comment (RLRabinowitz) [inline lambda.tf:85]: We'd need this variable for the other lambda as well

**Review by RLRabinowitz — APPROVED** [review]

File: lambda.tf (hunk 2 above)
Comment (Yantrio) [reply lambda.tf:85]: I can add that to my PR when it's needed :D

**Pull request merged**